### ISSUE TITLE
Provide support for regulatory sites such as transcription binding sites, promoters 

### ISSUE THREAD

**Issue by nathandunn** [GMOD/Apollo#32]
This will be done similar to #524 (and is dependent on it)  . . . at least initially.  We need visualizations for it later that imply a different relationship, as well as new data-types and SO terms.

**Comment by nathandunn** [comment]
Maybe related to #1955

**Comment by nathandunn** [comment]
I think for this we have to:
- [ ] annotate with / without evidence?
- [ ] relate directly to an isoform
- [ ] provide metadata info similar to other regions

**Comment by jrr-cpt** [comment]
Would addition of support for promoter features require a relationship to a gene or transcript? We are unlikely to have that data in many cases. However, for some well-studied phages, the promoter boundaries and transcripts are known.

**Comment by nathandunn** [comment]
@jrr-cpt that is good to know.  I'm sure we could have them in isolation.  We haven't really thought through the relationship types, so if you have any you explicitly wanted support for that would be helpful.

**Comment by nathandunn** [comment]
I think that everything is "is_a" or "part_of" if I recall, but it really depends on how you are exporting it.  For a typical GFF3, I'm not sure if it will matter, but potentially that is information we could add to the column 9 metadata.

**Comment by jrr-cpt** [comment]
IF we were to want to annotate transcipts as feature types, likely we would use [transcript](http://www.sequenceontology.org/browser/current_release/term/SO:0000673) and some of its children, like [polycistronic transcript](http://www.sequenceontology.org/browser/current_release/term/SO:0000078)

**Comment by nathandunn** [comment]
Yeah, if you have a list of the transcript types from the SO you want, that would be useful.    Attaching promoters to transcripts would have to come in a major new version of Apollo.  Adding types and non-coding promoters with no associations would be easier.
